xfce-extra/xfce4-panel-profiles/xfce4-panel-profiles-1.0.14.ebuild

```diff
@@ -19,14 +19,11 @@ SRC_URI="
 LICENSE="GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64 ~riscv ~x86"
-IUSE=""
+
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
-BDEPEND="
```

Comment:
This whole `*DEPEND` change doesn't seem to make any sense to me.

Reply:
Yes, please. We don't enforce a particular order of `*DEPEND` variables since they often cross-reference one another, so it's more efficient to use whichever order works best.